src/commands.ts

```diff
@@ -982,6 +981,70 @@ export const insertContributors = (
   return true
 }
 
+export const insertAward = (
+  state: ManuscriptEditorState,
+  dispatch?: Dispatch,
+  view?: EditorView
+) => {
+  if (dispatch) {
+    insertAwards(state, dispatch)
+    if (getChildOfType(state.doc, schema.nodes.awards, true)) {
+      const attrs = {
+        id: '',
+        source: '',
+        recipient: '',
+        code: '',
+      } as AwardAttrs
+      const awards = findChildrenByType(state.doc, schema.nodes.awards)[0]
+      const award = schema.nodes.award.create(attrs) as AwardNode
+      const pos = awards.pos + awards.node.nodeSize - 1
+      const tr = state.tr.insert(pos, award)
+      const selection = NodeSelection.create(tr.doc, pos)
+      if (view) {
+        view.focus()
+      }
+      dispatch(tr.setSelection(selection).scrollIntoView())
+    }
+  }
+
+  return true
+}
+
+export const insertAwards = (
```

Comment:
Can we move this to `doc.ts`? I think we should make it similar to `insertSupplementsNode` in that file (i.e. takes a transaction and returns the node).

Reply:
Done